src/app.js

```diff
@@ -1 +1,123 @@
-console.log('Hallo daar!');
\ No newline at end of file
+const api_url = "https://restcountries.com/v2/all";
+fetch(api_url)
+    .then((response) => {
+        return response.json();
+    })
+    .then((data) => {
+        let countryData = data
+        let lastSearch;
+        const countryList = document.createElement("ul");
+        document.getElementById("listContainer").appendChild(countryList);
+
+        function checkRegion(region) {
+            switch (region) {
+                case "Africa":
+                    return "blue"
+                case "Asia":
+                    return "red"
+                case "Europe":
+                    return "yellow"
+                case "Americas":
+                    return "green"
+                case "Oceania":
+                    return "purple"
+            }
+        }
+
+        const worldMap = document.createElement("img");
+        worldMap.src = new URL("assets/world_map.png", import.meta.url)
+        worldMap.setAttribute("id", "worldMap")
+        document.getElementById("imgContainer").appendChild(worldMap)
+
+        countryData.sort((a, b) => {
+            return a.population - b.population;
+        });
+
+        countryData.map(c => {
+            const countryData = document.createElement("li");
+            const contentContainer = document.createElement("dd")
+            const populationContainer = document.createElement("dd")
+            const content = document.createTextNode(c.name + " ");
+            const population = document.createTextNode("Has a population of " + c.population + " people")
+            const imgContainer = document.createElement("dd")
+            const img = document.createElement("img");
+            img.src = c.flag;
+            img.setAttribute("class", "flags")
+            contentContainer.appendChild(content);
+            populationContainer.appendChild(population);
+            imgContainer.appendChild(img)
+            countryData.appendChild(contentContainer)
+            countryData.appendChild(populationContainer)
+            countryData.appendChild(imgContainer)
+            countryData.setAttribute("class", checkRegion(c.region))
+            countryList.appendChild(countryData);
+        })
+//----------------------------------------------------------------------------------------------//
+        const displayCountry = (countrys) => {
```

Comment:
Je zou hier nog gebruik kunnen maken van spreading, zodat je niet overal `countrys.` voor hoeft te zetten
```suggestion
        const displayCountry = ({ name, currencies, population, region, capital }) => {
```